docs/software/apple/usage.mdx

```diff
@@ -3,6 +3,71 @@ id: usage
+
+The __Mesh Map__ will show all nodes on a map, nicely showing you the area of your reach. Some nodes do not broadcast their location, so you might miss them on the map.
+
+In __Settings__, you can change the configuration of your nodes. Most features are documented here https://meshtastic.org/docs/configuration
+
+In __Settings|AppSettings__, the Name is used when you use the request location feature in a channel or DM. "Block Range Test" will filter out range test messages that you might not want to be shown.
+
+In __Settings|Share Channel QR Code__, a QR code is shown that contains all channels; so after each change in the channel settings, the QR code should be distributed to all nodes you administer.
+
+In __Settings|User__, the "Licensed Operator" switch enables a mode for licensed HAM radio amateurs (may be dependent on the country you live in).
+
+In __Settings|Channels__, channels, which is something like user groups, can be added or deleted. The Channel Role has to be Secondary. The key size should always be 256 bit - all else is too easy to decipher. What you see here is the base64-representation of the PSK. The maximum number of channels in Meshtastic is 8. One can be the admin channel for remote configuration of your nodes. 
+
+__Settings|Bluetooth__ should be activated and the pairing Mode set to random - for best security. This will show the 6-digit Number when the node is paired with a new client.
+
+Serial Console and Debug Log can be enabled in __Settings|DeviceConfiguration|Device__. This does not log anything into the Apple app, but over a serial connection on the device if you have (soldered) one.
+
+## Reset and Power Off
+
+The node can be reset or powered off by the app: On the "Nodes" page, from the list choose the node that is connected via BLE, it will have 2 extra buttons "Reset" and "Power Off" in the node details.
+
+## Configuring Other Nodes
+
+Your other nodes can be configured from the Settings page. Cool!
+For this, you set up a new channel in __Settings|Channels__. The channel name has to be "admin" and it shall be a secondary channel. The other nodes need your channel QR code now; this is in __Settings|Share Channel QR Code__. NB: If you want to be safe, you cannot use email to send the QR code or the link.  
+You can now "Settings|Configure|Request Admin" for each of the other nodes on this channel. You can now fully configure your other nodes plus on page "Nodes" in the node list your nodes which are in your admin channel will have "Power Off" and "Reset" buttons in the node details now - just like the one that is connected dirctly via BLE. More info: https://meshtastic.org/docs/configuration/remote-admin
+
+## not with Apple
+
```

Comment:
I get that it could be useful to know some limitations of the app, but this list doesn't feel helpful.  I'd leave this section out.